.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "test/spec"]
 	path = test/spec
 	url = https://github.com/WebAssembly/testsuite.git
-[submodule "test/experimental"]
-	path = test/experimental
+[submodule "experimental"]
```

Comment:
I'll move that to https://github.com/WebAssembly/waterfall soon, I just need a mirror in the buildbots to prevent flakes (waiting on Chrome infra to do this).